simpler_faq/models.py

```diff
@@ -11,14 +13,17 @@ class Meta:
         verbose_name_plural = _('Topics')
         ordering = ['order']
 
-    def __unicode__(self):
-        return u'({0}) {1}'.format(self.order, self.text)
+    def __str__(self):
+        return '({0}) {1}'.format(self.order, self.text)
 
 
+@python_2_unicode_compatible
 class Question(models.Model):
     text = models.CharField(max_length=200)
     answer_text = models.TextField()
-    topic = models.ForeignKey(Topic, related_name='questions')
+    topic = models.ForeignKey(
+        Topic, related_name='questions', on_delete=models.DO_NOTHING
```

Comment:
Im not sure if `DO_NOTHING` is the best option, since `CASCADE` could erase the related `Topic` info